src/utils.ts

```diff
@@ -1,5 +1,9 @@
 const isNode = (): boolean => {
-  return typeof process !== 'undefined' && process.release.name === 'node'
+  return (
+    typeof process !== 'undefined' &&
+    typeof process.release !== 'undefined' &&
+    process.release.name === 'node'
```

Comment:
This `typeof x !== 'undefined` seems a bit weird to me.

Can we not do optional chaining like `process?.release?.name === node`?

Reply:
@hbergren 🤦‍♂ Yeah, much better lol